runtime/devnet/build.rs

```diff
@@ -1,12 +1,15 @@
-#[cfg(feature = "std")]
+#[cfg(all(feature = "std", feature = "metadata-hash"))]
 fn main() {
-	substrate_wasm_builder::WasmBuilder::new()
-		.with_current_project()
-		.export_heap_base()
-		.import_memory()
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash("PAS", 10)
```

Comment:
Because pop-network uses Paseo, it would make more sense to update this to `PAS 10` instead of `UNIT 12`

Reply:
Is this the metadata of the native token?

Reply:
Yes for the native token of the chain. 

Suggested by Ale: https://github.com/r0gue-io/pop-node/pull/194#pullrequestreview-2242457796

Reply:
Oke thank you!